ddb-dm-screen.user.js

```diff
@@ -7,246 +7,201 @@
 // @match        https://www.dndbeyond.com/campaigns/*
 // @grant        none
 // ==/UserScript==
+//
 var $ = window.jQuery;
+
+class Character {
+  constructor(json) {
+    this.json = json;
+    this.stats = new Object();
+
+    this.level = 0;
+    for (var i= 0; i < json.classes.length; i++) {
+        this.level += json.classes[i].level;
+    };
+    this.proficiency = Math.ceil(this.level / 4) + 1;
+
+    var stat = ['str', 'dex', 'con', 'int', 'wis', 'cha'];
+    for(var i = 0; i < 6; i++){
+      if (json.overrideStats[i].value == null) {
+        this.stats[stat[i]] = new Stat(json.stats[i].value + json.bonusStats[i].value, this.proficiency);
+      } 
+      else {
+        this.stats[stat[i]] = new Stat(json.overrideStats[i].value, this.proficiency);
+      }
+    }
+
```

Comment:
This if/else can be done in one line...
this.stats[stat[i]] = new Stat(json.overrideStats[i].value || (json.stats[i].value + json.bonusStats[i].value), this.proficiency);